Sources/NIOCore/FileHandle.swift

```diff
@@ -67,7 +67,7 @@ extension UInt64 {
         }
     }
 }
-#elseif arch(arm) || arch(i386) || arch(arm64_32)
```

Comment:
If this only matters the pointer size, `_pointerBitWidth(_64)` and `_pointerBitWidth(_32)` might be better choice

Reply:
+1, otherwise it misses plenty of other 32-bit architectures that in theory could be supported in the future.

Reply:
That's up to the NIO folks but
- `_pointerBitWidth*` is underscored and we have non-underscored alternatives (but yes, they're annoying)
- mixed platforms like `arm64_32` need to fall into the correct place, which in this case would be true (assuming that `_pointerBitWidth(_32)` is true for `arm64_32`

Reply:
My gut feeling is to avoid the underscored APIs. If it becomes necessary to support more 32-bit architectures in the future then we should push to get a non-underscored API.

Reply:
Sounds like the consensus is stick with ` || arch(wasm32)`. I'll leave as is unless someone voice opinions otherwise. Happy to change this if desired. I tried out adding ` || _pointerBitWidth(_32)` instead of ` || arch(wasm32)`, and that also worked, just as a data point.

Reply:
Friendly ping @glbrntt to approve & merge